model/model_training/configs/config.yaml

```diff
@@ -223,16 +206,39 @@ pythia-6.9B:
   per_device_train_batch_size: 4
   per_device_eval_batch_size: 4
 
-pythia-12B:
-  learning_rate: 6e-6
+pythia-12b:
```

Comment:
best would probably be to leave the pythia-12B config unchanged...

Reply:
i'm pretty sure the unchanged config is rather out of date, but this probably isn't the place to update it